Example/watchOSSample/SampleWatchApp.xcodeproj/project.pbxproj

```diff
@@ -16,8 +16,6 @@
 		5156949123F4965100E7C350 /* NotificationController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5156949023F4965100E7C350 /* NotificationController.swift */; };
 		5156949323F4965100E7C350 /* ComplicationController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5156949223F4965100E7C350 /* ComplicationController.swift */; };
 		5156949523F4965200E7C350 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 5156949423F4965200E7C350 /* Assets.xcassets */; };
-		515694AF23F4A3D200E7C350 /* NotificationService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 515694AE23F4A3D200E7C350 /* NotificationService.swift */; };
```

Comment:
Please revert this, this will remove messaging functions.

Reply:
The modification of watchOSSample in Example was made by mistake when I searched for reference. I will discard all the changes in this file.